src/definitions/chakra/theme/index.ts

```diff
@@ -2,7 +2,7 @@ import { extendTheme } from "@chakra-ui/react";
 
 const theme = extendTheme({
     fonts: {
-        body: "Roboto",
+        body: "Poppins",
```

Comment:
I believe @mahadzaryab1 added Roboto font. Which one should we use? For the one that's not used, let's remove the dependencies.

Reply:
The Figma uses Poppins